README.md

```diff
@@ -0,0 +1,168 @@
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+ Snapscan
+ 
+ - [📗 Table of Contents](#-table-of-contents)
+- [📖 Snapscan ](#-snapscan-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+    
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Snapscan <a name="about-project"></a>
+
+
+A mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+
+<div align="center">
+  <img src="./app/assets/images/diagram.png" alt="logo" />
+  <br/>
+</div>
```

Comment:
- [ ] Please this link is a broken link in your readme file so make sure that you remove or fix this link to make your README file professional :pray: